go/test/endtoend/onlineddl/vrepl_stress/onlineddl_vrepl_mini_stress_test.go

```diff
@@ -157,6 +160,18 @@ func nextOpOrder() int64 {
 	return opOrder
 }
 
+func TestInitialSetup(t *testing.T) {
+	repo, ok := os.LookupEnv("GITHUB_REPOSITORY") // `ok` tells us the env variable exists, hence that we are running in GitHub CI.
```

Comment:
If you do decide to stick with this, the `CI` env variable is IMO the right one to use here: https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables

We use that in at least one other spot. If that's too general, then a more Actions specific one would be `GITHUB_ACTION_REPOSITORY`.

Reply:
`CI` is too broad and does not indicate that this is a GitHub CI, where we're specifically looking to identify if this is a GitHub CI and which specific repository it is.

`GITHUB_ACTION_REPOSITORY` is actually **not** what we are looking for. It will not tell you `vitessio/vitess`. It will tell you, e.g. `actions/checkout`, i.e. the name of the repository of the _action_ code (a-la `import <url>` path). What we want to know is the name of the vitess repository this actions runs _on_.

`GITHUB_REPOSITORY` is therefore the most (and only?) correct variable that identifies where the GitHub CI is running on.

Reply:
Strike most of the above, given the suggestion of using `vCPUs`. I'll ignore the GitHub repository name altogether.

Reply:
Yep, strike all of the above! The `vCPU` based approach is 💯